src/trading/place-trade.js

```diff
@@ -28,34 +28,24 @@ var constants = require("../constants");
  * @param {function} p.onSuccess Called when the full trade completes successfully.
  * @param {function} p.onFailed Called if any part of the trade fails.
  */
+
 function placeTrade(p) {
   var normalizedPrice = normalizePrice({ minPrice: p.minPrice, maxPrice: p.maxPrice, price: p.limitPrice });
   var numTicks = new BigNumber(p.maxPrice, 10).minus(new BigNumber(p.minPrice, 10)).dividedBy(new BigNumber(p.tickSize, 10)).toFixed();
+  var orderType = (["buy", "sell"])[p._direction];
   getBetterWorseOrders({
     marketID: p._market,
     outcome: p._outcome,
-    amount: p.amount,
+    orderType,
     price: normalizedPrice,
```

Comment:
Unless `getBetterWorseOrders` denormalizes the price in augur-node, this should be `price: p.limitPrice` instead of `price: normalizedPrice`.